### DISCUSSION TITLE
GeneExpressionMatrix : Is it scaled or normalised ?

### DISCUSSION THREAD

**Discussion by annrosebright** [GreenleafLab/ArchR#1617] [Questions / Documentation]
Hi,

I am using ArchR for Multiome data. When I plot the expression of some genes using the plot embedding function, I find them very sparse.  Especially for genes that I know are abundantly present in the cell types, I am looking into. Usually, for scRNA-seq, the count matrix is scaled and normalized before differential or other analysis. So I was wondering if the gene expression matrix of the Multiome data also needs to process before doing find markers and visualization?   If so, what is the correct way to do it in ArchR? 

Thank you. 

Best,
Ann
![Screenshot 2022-09-09 at 17 53 33](https://user-images.githubusercontent.com/13627744/189391786-74f0cbb0-36cb-4aeb-b88c-f8b3a3248775.png)

**Comment by rcorces** [comment]
`GeneExpressionMatrix` represents log2 normalized (based on seq depth) counts.
You can find the relevant code here:
https://github.com/GreenleafLab/ArchR/blob/f6c0388bd37023400794c9ae8562ad69e3ba9fd7/R/MatrixGeneExpression.R#L257

**Reply by ginnyintifa** [reply]
Hi
![Screen Shot 2023-11-21 at 10 34 39 AM](https://github.com/GreenleafLab/ArchR/assets/13009028/683d93c4-7f5b-42bb-a8a9-00c9516ded65)
 I am also try to explore the same questions.

I found if I take colSum of my GeneExpressionMatrix, all the columns sum up to 10000, so I don't think this matrix represents log2 values. It could be the original count divided by sum and multiplied by 10000.
